### PR TITLE
feat: add metrics to disabled access log

### PR DESCRIPTION
**Pull request by mloiseleur**
##  Motivation 

Following https://github.com/ory/hydra/pull/3496, logs were reduce drastically.

The last unneeded logs are those generated when prometheus endpoint is scraped.

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I am following the
      [contributing code guidelines](../blob/master/CONTRIBUTING.md#contributing-code).
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got the approval (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added or changed [the documentation](https://github.com/ory/docs).

### CODE REVIEW

**Review by aeneasr — DISMISSED** [review]

**Review by bateller — APPROVED** [review]

**Comment by mloiseleur** [comment]
@aeneasr I needed to fix a linter issue, so it seems this PR needs another approval before it can be merged.

**Review by aeneasr — APPROVED** [review]

**Pull request merged**
